### DISCUSSION TITLE
Transfer amount is positive on both sides?

### DISCUSSION THREAD

**Discussion by edwardjamesgaff** [firefly-iii/firefly-iii#7040] [Firefly III] [answered]
When I create a transfer between two asset accounts, the amount shows up as positive on both sides. To rule out problems with my csv's and/or rules, I've tested it on the demo site by manually creating a simple transfer between 2 accounts. When I go to each account, the record for that transfer shows as positive regardless of whether money is leaving or entering the account.

I've found an issue below related to this which is marked as fixed:
https://github.com/firefly-iii/firefly-iii/issues/3382

I also found the following discussion from after the issue was fixed that seems to indicate that the issue still persists and suggests it is not intended to the fixed:
https://github.com/firefly-iii/firefly-iii/discussions/4674

I hope it is not intended behavior to have transfers away from an asset account show up as positive on that account. I have an account where the vast majority of the transactions are transfers into and away from it. Having both types of transfers show up as positive makes looking at my transaction history practically useless.

Apologies if I'm doing something incorrectly and please give me suggestions on how to do this properly.

**Comment by JC5** [comment, marked as answer]
The view will always be positive on both sides, yes. Sorry.

**Reply by edwardjamesgaff** [reply]
Thanks for clearing that up. I'm enjoying learning how to use Firefly III and really like a lot of its features.

However, I do hope you'll reconsider. I have to imagine that many people do something as simple as transfer money between checking and savings accounts, for example, and this issue makes tracking those transactions more confusing. But perhaps the lack of discussion around this proves I'm in the minority. 

In the meantime I'd love to hear if anyone has a workaround that makes transfers "make sense". Because at the moment for my use case I think the answer is "don't use transfers" which would be a shame.
